proxy/proxy_test.go

```diff
@@ -1247,6 +1248,77 @@ func TestFlusherImplementation(t *testing.T) {
 	}
 }
 
+func TestMeasureProxyWatch(t *testing.T) {
+	mo := metrics.Options{
+		Format:                     metrics.PrometheusKind,
+		EnableProxyRequestMetrics:  true,
+		EnableProxyResponseMetrics: true,
+	}
+	m := metrics.NewMetrics(mo)
+	defer m.Close()
+
+	doc := `*  -> latency("10ms") -> backendLatency("20ms") -> status(200) -> <shunt>`
+	tp, err := newTestProxyWithParams(doc, Params{Metrics: m})
```

Comment:
This is ok but it is preferable to use `proxytest.Config` (see https://github.com/zalando/skipper/pull/2220) see e.g. TestMetricsUncompressed